src/macos/macos_impl.rs

```diff
@@ -473,12 +465,9 @@ impl Enigo {
         self.held.clone()
     }
 
+    #[allow(clippy::unnecessary_wraps)]
     fn pressed_buttons() -> InputResult<usize> {
```

Comment:
No need to allow unnecessary wraps here. There is only one place where we call the function. It is not public and now is a one-liner. You can get rid of the function and replace where it gets called with that one line

Reply:
Fixed that!